.gitignore

```diff
@@ -20,6 +20,9 @@
 /npm-debug.log*
 /testem.log
 /yarn-error.log
+/.yalc*
+/yalc.lock
```

Comment:
Can revert once https://github.com/emberjs/ember-test-helpers/pull/1319 is released